include/SDL3/SDL_clipboard.h

```diff
@@ -147,7 +150,7 @@ extern DECLSPEC SDL_bool SDLCALL SDL_HasPrimarySelectionText(void);
  *
  * \sa SDL_SetClipboardData
  */
-typedef void *(SDLCALL *SDL_ClipboardDataCallback)(size_t *size, const char *mime_type, void *userdata);
+typedef void *(SDLCALL *SDL_ClipboardDataCallback)(void *userdata, const char *mime_type, size_t *size);
```

Comment:
Was this truly necessary? We'll have to intercept this in sdl2-compat. (same for `SDL_SetClipboardData`)

Reply:
This API isn't in SDL2, it's new for SDL 3.0.